point2.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Piza order</title>
+    <link rel="stylesheet" href="point2.css">
+  </head>
+  <body>
+    <form action="pizzaorder.html" target="_blank" method="get">
+      <label for="your_name">Your name:</label>
+      <input type="text" required name="your_name" maxlength="10" minlength="3" />
+      <br />
+
+      <label for="e-mail">Your e-mail:</label>
+      <input type="e-mail" required name="e-mail" placeholder="example@example.com" />
+      <br />
+      <br />
+      <label for="topping">Topping</label><br />
+      <input type="radio" name="topping" value="supreme" checked />Supreme
+      <input type="radio" name="topping" value="Vegetarian" />Vegetarian
+      <input type="radio" name="topping" value="Hawaian" />Hawaian
```

Comment:
Fiecare input trebuie sa fie legat de un label prin atributele for si id
```suggestion
      <p>Topping</p><br />
      <input type="radio" id="supreme" name="topping" value="supreme" checked /><label for="supreme">Supreme</label>
      <input type="radio" id="vegetarian" name="topping" value="Vegetarian" /><label for="vegetarian">Vegetarian</label>
      <input type="radio" name="topping" value="Hawaian" /><label for="hawaian">Hawaian</label>
```